Commands/Public/nuke.js

```diff
@@ -0,0 +1,71 @@
+module.exports = async ({ Constants: { Colors, Text }, client }, documents, msg, commandData) => {
+	if (msg.suffix) {
+		let num, query;
+		if (msg.suffix.trim().includes(" ")) {
+			num = msg.suffix.substring(0, msg.suffix.indexOf(" ")).trim();
+			query = msg.suffix.substring(msg.suffix.indexOf(" ") + 1).trim();
+		} else {
+			num = msg.suffix;
+		}
+
+		if (!num || isNaN(num) || (num !== -1 && num < 1) || num > 100) {
+			return msg.send({
+				embed: {
+					color: Colors.INVALID,
+					description: Text.INVALID_USAGE(commandData, msg.guild.commandPrefix),
+				},
+			});
+		}
+
+		num = parseInt(num);
+		let filter = () => true;
+		let before, after;
+
+		if (query) {
+			if (query.startsWith(":") && query.length > 1) {
```

Comment:
Do we need... These odd filters? They are super confusing, other than the member ones

Reply:
The before and after ones are probably not used enough to be necessary. The other 3 filters are pretty simple though: member, exact text, and `:<text>` for query text

Reply:
How about we change the shortcodes to something readable similar to Discord search? `before:<message ID> after:<message ID> from:<author> contains:"whatever string"`. Would also make it possible to include other filters such as using a date as before/after.

Reply:
Lets remove the before and after then

Reply:
Excuse me I use after all the time because I don't want to count every single message, I just nuke 99 until before my target message 🤔